src/rtcModule/webrtc.cpp

```diff
@@ -1840,7 +1983,7 @@ bool Session::cmd(uint8_t type, Args... args)
     {
         if (mState < kStateTerminating)
         {
-            asyncDestroy(TermCode::kErrNetSignalling);
+            mCall.destroy(TermCode::kErrNetSignalling, false);
```

Comment:
A fail in sendCommand is considerate a fatal error and there isn't any possibility to create a new Session then the call is finished directly. In previous code, session was destroyed asynchronous because we are in a session member method